src/scripts/createMarkdown.sh

```diff
@@ -58,12 +58,12 @@ This section provides a **high-level summary** of the key accessibility criteria
 
 More details: [Web Accessibility Checklist](https://www.magentaa11y.com/checklist-web/html/).
```

Comment:
Can we alter this line to be: 
Full information: https://www.magentaa11y.com/checklist-web/button/ 
^ is there any way to make the folder and the new component dynamic?